scripts/zkai_attention_kv_stwo_fine_grained_component_schema_gate.py

```
@@ -0,0 +1,762 @@
+
+
+def section_delta_rows() -> list[dict[str, Any]]:
+    payload = section_delta.build_payload()
+    return payload["profile_rows"]
+
+
+def artifact_specs(rows: list[dict[str, Any]]) -> list[dict[str, str]]:
+    specs = []
+    for row in rows:
+        for role in ROLES:
+            specs.append({"profile_id": row["profile_id"], "role": role, "path": row["artifacts"][role]["path"]})
```

Ruff (0.15.12)

[warning] Use `list.extend` to create a transformed list

(PERF401)